src/App.js

```diff
@@ -104,37 +118,53 @@ export default function App() {
     };
 
     return (
-        <div className={classes.root}>
-            {!isLoading ? (
-                <Router>
-                    <CssBaseline />
-                    <Header profile={profile} />
-                    <Switch>
-                        <Route exact path="/">
-                            <HoursContainer type="month" profile={profile} />
-                        </Route>
-                        <Route path="/template">
-                            <Typography
-                                variant="h4"
-                                component="h4"
-                                className={classes.title}
-                            >
-                                Template
-                            </Typography>
-                            <HoursContainer type="template" profile={profile} />
-                        </Route>
-                    </Switch>
-                </Router>
-            ) : (
-                <PreLoad />
-            )}
-
-            <script src="https://www.gstatic.com/firebasejs/7.11.0/firebase-app.js"></script>
-
-            <script src="https://www.gstatic.com/firebasejs/7.11.0/firebase-analytics.js"></script>
-
-            <script src="https://www.gstatic.com/firebasejs/7.11.0/firebase-auth.js"></script>
-            <script src="https://www.gstatic.com/firebasejs/7.11.0/firebase-firestore.js"></script>
-        </div>
+        <ThemeProvider theme={theme}>
+            <div className={classes.root}>
+                {!isLoading ? (
```

Comment:
if(isLoading) return null; Boven dit return statement is in mijn ogen netter en houdt de return schoner.